spark/src/main/scala/org/apache/spark/sql/comet/CometScanExec.scala

```diff
@@ -132,7 +132,7 @@ case class CometScanExec(
   lazy val bucketedScan: Boolean = wrapped.bucketedScan
 
   override lazy val (outputPartitioning, outputOrdering): (Partitioning, Seq[SortOrder]) =
-    (wrapped.outputPartitioning, wrapped.outputOrdering)
+    (UnknownPartitioning(wrapped.inputRDD.getNumPartitions), wrapped.outputOrdering)
```

Comment:
Btw, I think we should keep original partition instead a hard-coded `UnknownPartitioning`?

Reply:
I did this based on what we do in CometNativeScanExec as well as the original output partition. The original wrapped.outputPartitioning for CometScanExec inherits from FileSourceScanLike and this returns a hardcoded `  UnknownPartitioning(0)` for non-bucketed scan.

Reply:
Okay